tywrap_ir/tests/test_ir_async_generator_flags.py

```
@@ -0,0 +1,45 @@
+async def sample_async_coroutine():
+    return 1
+
+
+def sample_sync_generator():
+    yield 1
+
+
+class IRFunctionFlagTests(unittest.TestCase):
+    def test_async_generator_is_marked_as_generator(self) -> None:
+        result = _extract_function(sample_async_generator, "tests.sample_async_generator")
+        self.assertIsNotNone(result)
```

Ruff (0.15.6)

[warning] Use a regular `assert` instead of unittest-style `assertIsNotNone`

Replace `assertIsNotNone(...)` with `assert ...`

(PT009)